sympy/liealgebras/SU.py

```diff
@@ -0,0 +1,74 @@
+from sympy.matrices import zeros, ones, Matrix, randMatrix, matrices
+from sympy import mpmath as mp
+from mpmath import qr_solve
+from mpmath import lu_solve
+
+#generator of SU(n) group
+#now works only for SU(2), but since all the other functions can be
+#used for every SU(n) group, i will have to write only a new generator
+#here
+
+def SU_generator(n,str):
```

Comment:
Call the argument something other than `str`. `str` is a builtin name in Python and it's bad to shadow those.